stl/inc/xmemory

```diff
@@ -781,6 +781,33 @@ struct _Simple_types { // wraps types from allocators with simple addressing for
     using const_pointer   = const value_type*;
 };
 
+#if _HAS_CXX23
+// STRUCT TEMPLATE allocation_result

```

Comment:
We should really decide whether we want shouty banners in new code

Reply:
AFAIK we currently add SHOUTY BANNERS in old headers and don't add in new ones. But I agree that we should finally resolve #306 (hopefully deciding on removing them)